Make.inc

```diff
@@ -1190,7 +1190,7 @@ endif
 
 # We need python for things like BB triplet recognition.  We don't really care
 # about version, generally, so just find something that works:
-PYTHON := "$(shell which python 2>/dev/null || which python3 2>/dev/null || which python2 2>/dev/null || echo not found)"
+PYTHON := $(shell which python 2>/dev/null || which python3 2>/dev/null || which python2 2>/dev/null || echo not found)
```

Comment:
This broke things on cygwin:

```
$ make print-PYTHONmake: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory/home/Administrator/julia/Make.inc:1220: normalize_triplet.py appears to be non-functional 
(used python interpreter "/cygdrive/c/Program Files/Python38/python"), so BinaryBuilder disabled
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
/bin/sh: /cygdrive/c/Program: No such file or directory
PYTHON=/cygdrive/c/Program Files/Python38/python
```

To make things weirder we define this code twice:
First here https://github.com/JuliaLang/julia/blob/15244663a345a98e2919837fbd6d9e0ad997d5ba/Make.inc#L120 where we still wrap it in `"`

cc: @staticfloat